src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -29046,9 +29046,21 @@ struct S1
                 // (11,17): error CS0165: Use of unassigned local variable 'y1'
                 //         S1 z1 = y1;
                 Diagnostic(ErrorCode.ERR_UseDefViolation, "y1").WithArguments("y1").WithLocation(11, 17),
+                // (12,14): warning CS8600: Converting null literal or possible null value to non-nullable type.
```

Comment:
>// (12,14): warning CS8600: Converting null literal or possible null value to non-nullable type. [](start = 16, length = 96)

This and other warnings added in this unit-test look like a cascading diagnostics. Intentional?

Reply:
Yes, this is a consequence of the change.

---
In reply to: [284929567](https://github.com/dotnet/roslyn/pull/35765#discussion_r284929567) [](ancestors = 284929567)

Reply:
> Yes, this is a consequence of the change.

Was it intentional?

---
In reply to: [284985684](https://github.com/dotnet/roslyn/pull/35765#discussion_r284985684) [](ancestors = 284985684,284929567)

Reply:
Not intentional, but the update behavior seems reasonable.

Logged [https://github.com/dotnet/roslyn/issues/35790](https://github.com/dotnet/roslyn/issues/35790) to consider removing the warning.

---
In reply to: [285213774](https://github.com/dotnet/roslyn/pull/35765#discussion_r285213774) [](ancestors = 285213774,284985684,284929567)